src/list/description-term/DescriptionTerm.tsx

```diff
@@ -0,0 +1,52 @@
+import "./_description-term.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+interface DescriptionTermProps {
+  testid?: string;
+  title: string;
+  description: React.ReactNode;
+  descriptionFirst?: boolean;

```

Comment:
This could be done with. What do you think?

```css
.description-term {
  display: flex;
  flex-direction: column-reverse;
}
```

or 

```css
.description-term {
  display: flex;
  flex-direction: column;
}

.description-term__title {
  order: 1;
}
```

Reply:
We can do this but maybe usage with prop have other benefits. Let's ask @jamcry's opinion about this.

Reply:
I think this was defined as a prop because with this, in the same list, there may be some elements with the default order, some element with `descriptionFirst`. I think this would be harder if we made it with css, I believe we would have to pass classnames to each term.

```js
{
   id: "a",
   descriptionFirst: true
},
{
   id: "b",
   descriptionFirst: false
},
{
   id: "c",
   descriptionFirst: true
},
```

would be something like this:

```js
{
   id: "a",
      customClassName: "xx-list__reversed-order-term"
},
{
   id: "b",
},
{
  id: "c",
     customClassName: "xx-list__reversed-order-term"
 }
```

**But**, I don't think this has many use cases. We almost always use with all of them `descriptionFirst: true` when needed. Maybe we can discuss this next week. (I am off next Thursday, but I vote for CSS version as well 😄 )